### PR TITLE
add vmware.md

### PR DESCRIPTION
**Pull request by mamadou-diallo**
Added content for vmware - fix for Issue #80

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Company data addition
# How Has This Been Tested?

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: companies/vmware.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+fortune_rank: 42
```
[the remark this answers is not in the document]
Comment (mamadou-diallo) [reply companies/vmware.md]: My mistake I just realized that they were ranked in the top 100 best company to work for and not in the fortune 50, that has been fixed.

File: companies/vmware.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+name: VMware
+tags: ["fortune500" , "tech"]
+blm_statements:
+  - url: https://twitter.com/vmwarenews/status/1267629184097988608
+    date_posted: 2020-06-01
+    type: Tweet
+    raw_text:
+      "There is no place in our world for racial injustice.
```
[the remark this answers is not in the document]
Comment (mamadou-diallo) [reply companies/vmware.md:9]: I'm having a little trouble running eleventy on my PC and mac but I will try and have this resolved tomorrow !

[the remark this answers is not in the document]
Comment (kelseyhuse30) [reply companies/vmware.md:9]: No worries, thanks for the update @mamadou-diallo ! If you're having any specific errors you want help with, let me know.

[the remark this answers is not in the document]
Comment (mamadou-diallo) [reply companies/vmware.md:9]: Hello @kelseyhuse30, so I resolved the issues with running eleventy on my PC and can now run it locally... I'm trying to input this raw text for this cnbc article but keep receiving this : "YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key at line 52, column 33:" Error ... Could you maybe point towards the right direction on how to fix this?

[the remark this answers is not in the document]
Comment (kelseyhuse30) [reply companies/vmware.md:9]: yep! So for the `raw_text` field, if it's multi-line, use the `>` character for line 1, and then put the statement on the next line and make sure it's all indented.

Check out [this resource](https://www.w3schools.io/file/yaml-multiline-strings/) and search for 'folded block styles with > character`.  You can also take a look at the `companies/anthem.md` file in our repo as a good example of how it's done.

[the remark this answers is not in the document]
Comment (mamadou-diallo) [reply companies/vmware.md]: Hello @kelseyhuse30,

Thank you for linking that resource, it was very helpful as the example gave me a hint that my my indentation was slightly off. 
I have completed your change request and commited the update.
Let me know if there is anything else ! I'll be more than happy to fix it

[the remark this answers is not in the document]
Comment (kelseyhuse30) [reply companies/vmware.md]: Looks great!! I'm going to merge this in :)

**Review by kelseyhuse30 — APPROVED** [review]

**Comment by kelseyhuse30** [comment]
@mamadou-diallo  I saved an image from their tweet and uploaded it.  The VMWare page is now live!
https://diversitycorp.co/companies/vmware/

**Pull request merged**